docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+# Cosmos Hub (Gaia) LCD API
+
+This document describes the API that is exposed by the specific LCD implementation of the Cosmos
+Hub (Gaia). Those APIs are exposed by a REST server and can easily be accessed over HTTP/WS
+connections. 
+
+The complete API is comprised of the sub-APIs of different modules. The modules in the Cosmos Hub
+(Gaia) API are:
+    * ICS0 (TendermintAPI)
+    * ICS1 (KeyAPI)
+    * ICS20 (TokenAPI)
+    * ICS21 (StakingAPI)
+    * ICS22 (GovernanceAPI)
+
+
+## ICS0 - TendermintAPI
+
+The TendermintAPI describes all the endpoints that a full node exposes in order to learn about the
+status of the network.
+
+### /node_info - GET
+
+url: /node_info, Method: GET
+
+Functionality: Get LCD node status
+
+Parameters: null
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+      "id": "992e24f5761b37de48536cecff0a0687937049a3",
+            "listen_addr": "10.0.2.15:46656",
+            "network": "test-chain-F0bln0",
+            "version": "0.19.7-dev",
+            "channels": "4020212223303800",
+            "moniker": "lhy-ubuntu",
+            "other": [
+                "amino_version=0.9.9",
+                "p2p_version=0.5.0",
+                "consensus_version=v1/0.2.2",
+                "rpc_version=0.7.0/3",
+                "tx_index=on",
+                "rpc_addr=tcp://0.0.0.0:46657"
+      ]
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot get the node info",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /syncing - GET
+
+url: /syncing, Method: GET
+
+Functionality: Check syncing status of the fullnode which is connecting with the LCD node
+
+Parameters: null
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "syncing":false
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot get the syncing info",
```

Comment:
it would be nice if the error message included a possible explanation as to why there is an error and proposed a solution.